query-coordinator/src/main/scala/com/socrata/querycoordinator/caching/SoQLAnalysisDepositioner.scala

```diff
@@ -61,7 +61,8 @@ object SoQLAnalysisDepositioner {
   private def depositionHint[ColumnId,Type](hint: Hint[ColumnId, Type]) = {
     hint match {
       case x@Materialized(_) => x.copy(NoPosition)
-      case x@UniqueOrder(_) => x.copy(NoPosition)
```

Comment:
Why was this removed?

Reply:
I accept Marc's comment and decide not implementing unique order via hint.